support-workers/src/test/scala/com/gu/support/workers/lambdas/SendAcquisitionEventSpec.scala

```diff
@@ -24,10 +24,6 @@ class SendAcquisitionEventSpec extends AsyncLambdaSpec with MockContext {
     sendEvent(sendAcquisitionEventPrintJson)
   }
 
-  "SendAcquisitionEvent" should "work with GW 6 for 6 input" taggedAs IntegrationTest in {
-    sendEvent(sendAcquisitionEventGWJson)
-  }
```

Comment:
should we not change this to be non 6 for 6 rather than deleting?  The print one is card and this one is/was paypal.

Reply:
I'll do this in the next PR which will remove the old promo fields

Reply:
Added back in https://github.com/guardian/support-frontend/pull/6414